[0m17:42:58.176000 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f001bd91cd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f001bd9f090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f001bf2ad90>]}


============================== 17:42:58.179119 | bc537161-96cf-4614-8039-6970f1c71075 ==============================
[0m17:42:58.179119 [info ] [MainThread]: Running with dbt=1.8.3
[0m17:42:58.181390 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/home/mantis/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m17:42:58.213471 [info ] [MainThread]: dbt version: 1.8.3
[0m17:42:58.214243 [info ] [MainThread]: python version: 3.11.0rc1
[0m17:42:58.214938 [info ] [MainThread]: python path: /home/mantis/airflow-dbt/.venv/bin/python
[0m17:42:58.215825 [info ] [MainThread]: os info: Linux-5.15.0-113-generic-x86_64-with-glibc2.35
[0m17:42:58.402374 [debug] [MainThread]: Spark adapter: Setting pyhive.hive logging to ERROR
[0m17:42:58.412158 [debug] [MainThread]: Spark adapter: Setting thrift.transport logging to ERROR
[0m17:42:58.412717 [debug] [MainThread]: Spark adapter: Setting thrift.protocol logging to ERROR
[0m17:42:59.658653 [info ] [MainThread]: Using profiles dir at /home/mantis/.dbt
[0m17:42:59.659701 [info ] [MainThread]: Using profiles.yml file at /home/mantis/.dbt/profiles.yml
[0m17:42:59.662435 [info ] [MainThread]: Using dbt_project.yml file at /home/mantis/airflow-dbt/dbt_project.yml
[0m17:42:59.666327 [info ] [MainThread]: adapter type: databricks
[0m17:42:59.666839 [info ] [MainThread]: adapter version: 1.8.3
[0m17:42:59.669938 [info ] [MainThread]: Configuration:
[0m17:42:59.670495 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m17:42:59.676842 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m17:42:59.678214 [info ] [MainThread]: Required dependencies:
[0m17:42:59.678804 [debug] [MainThread]: Executing "git --help"
[0m17:42:59.684899 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m17:42:59.685658 [debug] [MainThread]: STDERR: "b''"
[0m17:42:59.686225 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m17:42:59.686662 [info ] [MainThread]: Connection:
[0m17:42:59.687220 [info ] [MainThread]:   host: teste
[0m17:42:59.687919 [info ] [MainThread]:   http_path: teste
[0m17:42:59.688477 [info ] [MainThread]:   catalog: db-rs
[0m17:42:59.688861 [info ] [MainThread]:   schema: db
[0m17:42:59.689414 [info ] [MainThread]: Registered adapter: databricks=1.8.3
[0m17:42:59.699099 [debug] [MainThread]: Databricks adapter: DatabricksDBTConnection(id=139638226369744, session-id=None, name=debug, idle-time=0s, acquire-count=0, language=None, thread-identifier=(8791, 139638695972864), compute-name=) - Creating connection
[0m17:42:59.699724 [debug] [MainThread]: Acquiring new databricks connection 'debug'
[0m17:42:59.700136 [debug] [MainThread]: Databricks adapter: DatabricksDBTConnection(id=139638226369744, session-id=None, name=debug, idle-time=6.4373016357421875e-06s, acquire-count=1, language=None, thread-identifier=(8791, 139638695972864), compute-name=) - Acquired connection on thread (8791, 139638695972864), using default compute resource
[0m17:42:59.700558 [debug] [MainThread]: Databricks adapter: DatabricksDBTConnection(id=139638226369744, session-id=None, name=debug, idle-time=0.0004513263702392578s, acquire-count=1, language=None, thread-identifier=(8791, 139638695972864), compute-name=) - Checking idleness
[0m17:42:59.700940 [debug] [MainThread]: Databricks adapter: DatabricksDBTConnection(id=139638226369744, session-id=None, name=debug, idle-time=0.0008299350738525391s, acquire-count=1, language=None, thread-identifier=(8791, 139638695972864), compute-name=) - Retrieving connection
[0m17:42:59.701411 [debug] [MainThread]: Using databricks connection "debug"
[0m17:42:59.701757 [debug] [MainThread]: On debug: select 1 as id
[0m17:42:59.702084 [debug] [MainThread]: Opening a new connection, currently in state init
[0m17:50:57.470664 [debug] [MainThread]: Databricks adapter: DatabricksDBTConnection(id=139638226369744, session-id=None, name=debug, idle-time=1.3828277587890625e-05s, acquire-count=0, language=None, thread-identifier=(8791, 139638695972864), compute-name=) - Released connection
[0m17:50:57.471728 [error] [MainThread]: Encountered an error:

[0m17:50:57.645267 [error] [MainThread]: Traceback (most recent call last):
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/socket.py", line 961, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
                       ^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f000ede6690>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/cli/main.py", line 406, in debug
    results = task.run()
              ^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/task/debug.py", line 150, in run
    connection_status = self.test_connection()
                        ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/task/debug.py", line 471, in test_connection
    connection_result = self.attempt_connection(self.profile)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/task/debug.py", line 449, in attempt_connection
    adapter.debug_query()
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/spark/impl.py", line 508, in debug_query
    self.execute("select 1 as id")
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/utils.py", line 78, in wrapper
    return func(*new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/impl.py", line 226, in execute
    return super().execute(sql=sql, auto_begin=auto_begin, fetch=fetch, limit=limit)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/base/impl.py", line 350, in execute
    return self.connections.execute(sql=sql, auto_begin=auto_begin, fetch=fetch, limit=limit)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/connections.py", line 629, in execute
    _, cursor = self.add_query(sql, auto_begin)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/connections.py", line 600, in add_query
    cursor = cast(DatabricksSQLConnectionWrapper, connection.handle).cursor()
                                                  ^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/contracts/connection.py", line 95, in handle
    self._handle.resolve(self)
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/contracts/connection.py", line 119, in resolve
    return self.opener(connection)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/connections.py", line 1080, in open
    return cls.retry_connection(
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/base/connections.py", line 237, in retry_connection
    connection.handle = connect()
                        ^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/dbt/adapters/databricks/connections.py", line 1044, in connect
    conn = dbsql.connect(
           ^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/__init__.py", line 90, in connect
    return Connection(server_hostname, http_path, access_token, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/client.py", line 233, in __init__
    self._open_session_resp = self.thrift_backend.open_session(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/thrift_backend.py", line 578, in open_session
    response = self.make_request(self._client.OpenSession, open_session_req)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/thrift_backend.py", line 495, in make_request
    response_or_error_info = attempt_request(attempt)
                             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/thrift_backend.py", line 405, in attempt_request
    response = method(request)
               ^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/thrift_api/TCLIService/TCLIService.py", line 204, in OpenSession
    self.send_OpenSession(req)
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/thrift_api/TCLIService/TCLIService.py", line 213, in send_OpenSession
    self._oprot.trans.flush()
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/databricks/sql/auth/thrift_http_client.py", line 177, in flush
    self.__resp = self.__pool.request(
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/request.py", line 81, in request
    return self.request_encode_body(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/request.py", line 173, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 827, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 827, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 827, in urlopen
    return self.urlopen(
           ^^^^^^^^^^^^^
  [Previous line repeated 9 more times]
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/connectionpool.py", line 802, in urlopen
    retries.sleep()
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/util/retry.py", line 432, in sleep
    self._sleep_backoff()
  File "/home/mantis/airflow-dbt/.venv/lib/python3.11/site-packages/urllib3/util/retry.py", line 416, in _sleep_backoff
    time.sleep(backoff)
KeyboardInterrupt

[0m17:50:57.665500 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 479.55157, "process_user_time": 2.351271, "process_kernel_time": 1.180357, "process_mem_max_rss": "212128", "process_in_blocks": "12768", "process_out_blocks": "48", "command_success": false}
[0m17:50:57.666293 [debug] [MainThread]: Command `dbt debug` failed at 17:50:57.666157 after 479.55 seconds
[0m17:50:57.666829 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m17:50:57.667495 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f002abf3b10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f001bdbce10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f001bd53e10>]}
[0m17:50:57.668003 [debug] [MainThread]: Flushing usage events
